packages/delisp-core/src/reader.ts

```diff
@@ -164,6 +166,20 @@ const sexpr: Parser<ASExpr> = spaced(
   reportUnmatched.then(atom.or(() => list(sexpr)))
 );
 
+const sexprs: Parser<ASExpr[]> = until(endOfInput, sexpr);
+
+export function readAllFromString(str: string): ASExpr[] {
+  const result = sexprs.parse(str);
```

Comment:
Can we extract this logic somewhere?  IT's shared with `readFromString`.  Maybe we should just have a `.parseFromString` method in the parsers combinators file?